### PR TITLE
DT-391 Split local api from core-api

### PR DESCRIPTION
**Pull request by ianaz**

### CODE REVIEW

File: docs/.vitepress/config.js [hunk 1]
```diff
@@ -149,6 +149,10 @@ export default defineConfig({
               text: "Local API",
               link: "/references/api",
             },
+            {
+              text: "Core API",
```
Comment (emeka) [inline docs/.vitepress/config.js:153]: Maybe Admin API?

File: docs/references/openapi-core.yaml [hunk 2]
```diff
@@ -0,0 +1,346 @@
+openapi: "3.0.2"
+
+info:
+  title: Certifaction Core API
```
Comment (emeka) [inline docs/references/openapi-core.yaml:4]: Maybe Certifaction Admin API

File: docs/references/openapi-core.yaml [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+  description: |
+    Certifaction Core-API is the central management system for our digital signing platform, enabling secure handling of users, organizations, roles and others.
```
Comment (emeka) [inline docs/references/openapi-core.yaml:6]: Certifaction Admin API is used to to manage users, roles and organisations.

**Review by emeka — APPROVED** [review]
Approved with comments

File: docs/references/openapi-core.yaml (hunk 3 above)
Comment (ianaz) [reply docs/references/openapi-core.yaml]: [3a83b9d](https://github.com/certifaction/cli/pull/44/commits/3a83b9d2f47de997a9afbbf6ad291d1e523aa156)

File: docs/references/openapi-core.yaml (hunk 2 above)
Comment (ianaz) [reply docs/references/openapi-core.yaml]: [795083d](https://github.com/certifaction/cli/pull/44/commits/795083dd7089debc6662e5c60831c0fccffded56)

File: docs/.vitepress/config.js (hunk 1 above)
Comment (ianaz) [reply docs/.vitepress/config.js]: [795083d](https://github.com/certifaction/cli/pull/44/commits/795083dd7089debc6662e5c60831c0fccffded56)

**Pull request merged**